docs/source/private/overview.rst

```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
 How will this be gathered?
```

Comment:
Lets make this clear and call it `Preparing results upload` or something similar.

Reply:
Changed this.